hack/dev-functions.sh

```diff
@@ -87,6 +87,13 @@ function local_argocd_login() {
 		echo "ARGOCD_LOCAL_ADDRESS=https://localhost:$argoserver_nodeport"
 	} >.env.local
 
+	if [ ! -d .prcconfig ]; then
```

Comment:
This sets up a config folder locally so we can `make run` and use it to connect to the `argocd-server` in `kind`.